src/Ubiquity.NET.Llvm/Metadata/ModuleFlag.cs

```diff
@@ -34,33 +30,5 @@ public ModuleFlag( ModuleFlagBehavior behavior, string name, LlvmMetadata metada
 
         /// <summary>Gets the Metadata for this flag</summary>
         public LlvmMetadata Metadata { get; }
-
```

Comment:
Note that this constructor appears to be dead code even in the original LLVM Ubiquity sources.